sdk/servicebus/azure-messaging-servicebus/pom.xml

```diff
@@ -42,7 +42,7 @@
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-core</artifactId>
-      <version>1.17.0</version> <!-- {x-version-update;unreleased_com.azure:azure-core;dependency} -->
```

Comment:
Can you rebase your changes on top of this main branch? These changes should not be here. In addition, you might want to update your local `main`.

https://git-scm.com/book/en/v2/Git-Branching-Rebasing

```sh
git checkout issue#20451
git rebase master
```

Reply:
According to your comment, fixed within new version